.buildkite/pipeline.yml

```diff
@@ -126,6 +126,15 @@ steps:
         key: "slabplanet_default"
         command: "julia --color=yes --project=experiments/AMIP/modular/ experiments/AMIP/modular/coupler_driver_modular.jl --run_name slabplanet_default --FLOAT_TYPE Float64 --coupled true --surface_setup PrescribedSurface  --moist equil --vert_diff true --rad gray --energy_check true --mode_name slabplanet --t_end 10days --dt_save_to_sol 9days --dt_cpl 200 --dt 200secs --mono_surface true --h_elem 4 --precip_model 0M --anim true --job_id slabplanet_default"
         artifact_paths: "experiments/AMIP/modular/output/slabplanet/slabplanet_default_artifacts/*"
+        agents:
+          slurm_mem: 20GB
```

Comment:
Why are we specifying the memory for each run now?

Reply:
Due to the 137 error. Following advice from the Atmos team.